src/main/python/api.py

```diff
@@ -6,7 +6,13 @@
 from flask import Flask, jsonify, request
 # FIXME: separate this out to a classifier class where we can switch out the models
 from pyserini import Pyserini
-from jaccard import Jaccard
+from sm_model.bridge import SMModelBridge
+
+path_to_castorini = os.getcwd() + "/../../../.."
```

Comment:
This requires the directory structure to be maintained so it's important that they clone the repos in the same place.